### PR TITLE
Mixpanel tracking improvements

### LINKED ISSUE #2698: Log to analytics the total transaction counts users on mainnet and testnet
Upon sending a transaction (confirmation via actionsheet, don't wait for block confirmation)

```
if testnet
    testnetTransactionCount += 1
else
    transactionCount += 1
```

Looks like this for iOS SDK:

```
mixpanelInstance.people.increment(property: "testnetTransactionCount", by: 1)
```

### PR DESCRIPTION
**Pull request by justindg**
- Closes #2698

Other improvements in this PR:
- Confirmation Dialog result was previously not tracked properly (only in `TokenFunctionActivity` and `SwapActivity`). Moved the implementation to `CreateTransactionInteract` to track every sign and send transaction. Partially closes items in #3028, as some properties require more tinkering to track.
- Created `UserProperties` object under `Analytics` to consolidate items

### CODE REVIEW

File: app/src/main/java/com/alphawallet/app/analytics/Analytics.java [hunk 1]
```diff
@@ -13,13 +13,34 @@
...
     public static final String PROPS_CUSTOM_NETWORK_SYMBOL = "symbol";
     public static final String PROPS_CUSTOM_NETWORK_IS_TESTNET = "is_testnet";
+    public static final String PROPS_TRANSACTION_CHAIN_ID = "chain";
+    public static final String PROPS_TRANSACTION_TYPE = "transactionType";
+    public static final String PROPS_TRANSACTION_SPEED_TYPE = "speedType"; // TODO
+    public static final String PROPS_TRANSACTION_IS_ALL_FUNDS = "isAllFunds"; // TODO
+
+    public enum UserProperties
+    {
+        TRANSACTION_COUNT("transactionCount"),
```
[the remark this answers is not in the document]
Comment (justindg) [reply app/src/main/java/com/alphawallet/app/analytics/Analytics.java:28]: The property names are specified in #2698 

> The enum name and property is duplicated. Suggest using `TRANSACTION_COUNT.name().toLowerCase()`.

File: app/src/main/java/com/alphawallet/app/analytics/Analytics.java [hunk 2]
```diff
@@ -13,13 +13,34 @@
...
     public static final String PROPS_SWAP_TO_TOKEN = "to_token";
     public static final String PROPS_ERROR_MESSAGE = "error_message";
-
     public static final String PROPS_CUSTOM_NETWORK_NAME = "network_name";
     public static final String PROPS_CUSTOM_NETWORK_RPC_URL = "rpc_url";
     public static final String PROPS_CUSTOM_NETWORK_CHAIN_ID = "chain_id";
     public static final String PROPS_CUSTOM_NETWORK_SYMBOL = "symbol";
     public static final String PROPS_CUSTOM_NETWORK_IS_TESTNET = "is_testnet";
+    public static final String PROPS_TRANSACTION_CHAIN_ID = "chain";
+    public static final String PROPS_TRANSACTION_TYPE = "transactionType";
```
[the remark this answers is not in the document]
Comment (justindg) [reply app/src/main/java/com/alphawallet/app/analytics/Analytics.java:22]: The property names are specified in #3028

> The naming convention does not conform to the upper names.

**Review by JamesSmartCell — CHANGES_REQUESTED** [review]
Hi @justindg - all looks good code wise - could you rebase with master (not merge!) and update and see if the tests all pass before we merge the PR.

**Comment by justindg** [comment]
Moved to #3186

**Pull request closed without merging**